app/models.py

```diff
@@ -17,13 +17,24 @@ class Plant(db.Model):
     gardens = relationship("Garden", secondary="garden_plants")
 
     def get_all():
-        return Plant.query.all()
+        all_plants = Plant.query.all()
+        return all_plants
+
+    def get_by_id(id):
+        plant =  Plant.query.get_or_404(id)
+        return plant
 
     def random_id():
         from random import randint
-        num = randint(1, 24)
+        plants = Plant.query.all()
```

Comment:
We should try and make this into a more dynamic method that pull the whole record. Maybe we can try with the engine or func.rand() again tomorrow

Reply:
Good call, I forgot we even had that as a dynamic range!  Let's add it as a card and maybe we can make it work tomorrow.  It would definitely be nice to pull a random plant instead of have a set range of numbers that might, or might not be ids in the DB.